packages/store-sync/src/createStoreSync.ts

```diff
@@ -96,7 +97,7 @@ export async function createStoreSync<config extends StoreConfig = StoreConfig>(
       filters,
       initialState,
       initialBlockLogs,
-      indexerUrl,
+      indexerUrl: indexerUrl ?? (publicClient.chain as MUDChain).indexerUrl,
```

Comment:
Do we want to pass the chain more explicitly to avoid this cast?

Reply:
no, I don't want to have a potential deviation between publicClient's chain and a chain passed in

this cast is fine but you could be super defensive with something like
```
"indexerUrl" in publicClient.chain && typeof publicClient.chain.indexerUrl === 'string' ? publicClient.chain.indexerUrl : undefined
```

Reply:
we prob also still want some way to turn off the indexer if needed

maybe we also allow `indexerUrl: false` here?